CONTRIBUTING.rst

```diff
@@ -20,10 +20,11 @@ Pull Requests
 =============
 
 If you want to contribute to the RosettaSciIO source code, you can send us a
-`pull request <https://github.com/hyperspy/rosettasciio/pulls>`_. Small bug fixes
-are corrections to the user guide are typically a good starting point. But don't
-hesitate also for significant code contributions, such as support for a new
-file format - if needed, we'll help you to get the code ready to common standards.
+`pull request <https://github.com/hyperspy/rosettasciio/pulls>`_ against the ``main``
```

Comment:
While I was around there: I specified "against the `main` branch" to clarify https://github.com/hyperspy/rosettasciio/issues/97#issuecomment-1573518005.